docs/source/contributing/releasing.adoc

```diff
@@ -9,7 +11,7 @@ toc::[]
 [[releasing-overview]]
 == Overview
 
-The following sections describe how to release Minishift either via automated job or manually.
+The following sections describe how to release {project} either via automated job or manually.
```

Comment:
So here it does just not make sense anymore. This is really just about Minishift release, not CDK. TBH, I don't think we get away with a plain find&replace of all Minishift occurrences.